source/topics/paas/how-to-setup-application-monitoring.md

```diff
@@ -0,0 +1,9 @@
+Aptible recommends New Relic for application monitoring, and now supports it at the [host-level](https://github.com/aptible/support/issues/115) as well.
```

Comment:
This doesn't really explain how to set up Application Monitoring (APM), which is the title of the article. Most of this article explains how to set up Server Monitoring, which is probably not ideal, since the vast majority of useful information will come from APM, not Server Monitoring.